docs/source/releases/latest/852-remove-unused-sphinx-extensions.yaml

```diff
@@ -0,0 +1,18 @@
+removal:
```

Comment:
Do not merge this until https://github.com/NRLMMD-GEOIPS/geoips/pull/850 and https://github.com/NRLMMD-GEOIPS/geoips/pull/840 are merged - they fix the CI that is currently failing on this PR